pkg/tfgen/docs.go

```diff
@@ -201,9 +201,35 @@ func findRepoPath(repoPathsEnvVar string, moduleCoordinates string) string {
 	return ""
 }
 
+func getMarkdownNames(resourcePrefix, rawName string, globalInfo *tfbridge.DocRuleInfo) []string {
+	possibleMarkdownNames := []string{
+		// Most frequently, docs leave off the provider prefix
+		withoutPackageName(resourcePrefix, rawName) + ".html.markdown",
+		withoutPackageName(resourcePrefix, rawName) + ".markdown",
+		withoutPackageName(resourcePrefix, rawName) + ".html.md",
+		withoutPackageName(resourcePrefix, rawName) + ".md",
+		// But for some providers, the prefix is included in the name of the doc file
+		rawName + ".html.markdown",
+		rawName + ".markdown",
+		rawName + ".html.md",
+		rawName + ".md",
+	}
+
+	if globalInfo != nil && globalInfo.AlternativeNames != nil {
+		// We look at user generated names before we look at default names
+		possibleMarkdownNames = append(globalInfo.AlternativeNames(tfbridge.DocsPathInfo{
+			Resource: rawName,
```

Comment:
Doesn't this get called for data sources too?

Reply:
Yes it does.